IO_operations_C/binio.cpp

```diff
@@ -0,0 +1,48 @@
+//binary input and output integers
+#include<fstream>
+#include<iostream>
+using namespace std;
+const int MAX = 80;
+int buff[MAX];
+
+int main()
+{
+    for(int j=0;j<MAX;j++)
+    {
+        buff[j]=j;
+    }
+    ofstream os("edata.dat",ios::binary);
```

Comment:
can you add a sample file 'edata.dat' which contains some data, so that I can run the program across that data file.